src/detail-page/hooks/useTabData.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import { Redirect, useHistory, useRouteMatch } from 'react-router-dom';
+import { backend, devOps, frontend, projectManager, qa, uxUi } from '../../data/detail-page/tabs/index';
+import { Category } from '../../data/detail-page/tabs/tabsInterfaces';
+import { ProfessionKey } from '../../data/home-page/ITProfessions';
+import { JobsTab } from '../components/JobsTab';
+import { LearnTab } from '../components/LearnTab';
+import { ResourcesTab } from '../components/ResourcesTab';
+
+const getData = (professionId: ProfessionKey) => {
+  switch (professionId) {
+    case 'frontend':
+      return frontend;
+    case 'backend':
+      return backend;
+    case 'devops':
+      return devOps;
+    case 'projectmanager':
+      return projectManager;
+    case 'qa':
+      return qa;
+    case 'uxui':
+      return uxUi;
+    default:
+      return [];
+  }
+};
+
+export const useTabData = (professionId: ProfessionKey) => {
+  const [activeTab, setActiveTab] = useState(0);
+  const categories = getData(professionId);
+
+  const { path, url } = useRouteMatch();
```

Comment:
We're not using path or url anywhere inside the hook. Maybe we can simply remove it from the hook and use `useRouteMatch` directly in the components?

Reply:
We use url in `handleTabClicked` function, but we can also pass it as an argument ;)

Reply:
Oh, my bad, I missed it :) we can leave it here, then.

Reply:
Done :)